hooks/useGroupedTeamMemberData.tsx

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from "@tanstack/react-query";
+import { gql } from "@apollo/client";
+import { request } from 'graphql-request';
+
+export const groupedTeamMembersQuery = gql`
+  query groupedTeamMembersQuery {
```

Comment:
I'm going to use this to populate the "The Team" page. I'll add some types then.